puppet.go

```diff
@@ -297,6 +299,10 @@ func (puppet *Puppet) Sync() {
 		puppet.log.Debugln("No contact info found")
 	}
 
+	if puppet.UpdateContactInfo() {
+		puppet.Update()
+	}
```

Comment:
This should probably be in `SyncWithContact` among the other things needing `Update()` calls

Reply:
That was also my initial plan, but if it's in `SyncWithContact` then it won't get called if there's no contact object?

Reply:
It always gets called on the line below this change